Chat Server/app.js

```diff
@@ -7,23 +7,21 @@ const express = require('express');
 const cors = require('cors');
 const apiserver = express();
 const TINYPacket = require('./src/packets/Packet.js');
-const packets = require("./src/packets/Packet").packets;
-const connectPacket = require("./src/packets/Connect.js");
```

Comment:
I removed these because they weren't _apparently_ being used, but if they're passively being used (e.g., requiring Packet.js preloads 50 ketchup packets for delicious cheeseburgers), then we may wanna re-add.